common/lib/xmodule/xmodule/js/spec/video/sjson_spec.js

```diff
@@ -3,8 +3,10 @@ require(
 ['video/00_sjson.js'],
 function (Sjson) {
     describe('Sjson', function () {
-        var data = jasmine.stubbedCaption,
-            sjson;
+        var data = jasmine.stubbedCaption;
```

Comment:
It doesn't look like we're changing anything on these first two lines; can we keep them untouched?